### PR TITLE
Add jre headless subpackage

### PR DESCRIPTION
**Pull request by tuananh**
add jre-headless subpackage, addressing #142

### CODE REVIEW

File: openjdk-11.yaml [hunk 1]
```diff
@@ -112,6 +112,43 @@ subpackages:
       runtime:
         - freetype
         - fontconfig
+  - name: openjdk-11-jre-headless
+    description: "Oracle OpenJDK 17 (JRE headless)"
+    pipeline:
+      - runs: |
+          export _FROM_ROOT=${{targets.destdir}}/usr/lib/jvm/openjdk
+          export _TO_ROOT=${{targets.subpkgdir}}/usr/lib/jvm/openjdk
```
Comment (rawlingsj) [inline openjdk-11.yaml:120]: the JRE subpackage above is using jre in the folder path `/usr/lib/jvm/openjdk-jre` we should probably do the same here

File: Makefile [hunk 2]
```diff
@@ -191,7 +191,7 @@ $(eval $(call build-package,zip,3.0-r0))
 $(eval $(call build-package,libpng,1.6.38-r0))
 $(eval $(call build-package,freetype,2.12.1-r0))
 $(eval $(call build-package,fontconfig,2.14.0-r1))
-$(eval $(call build-package,openjdk-11,11.0.17.8-r2))
+$(eval $(call build-package,openjdk-11,11.0.17.8-r3))
 $(eval $(call build-package,openjdk-17,17.0.5.8-r2))
```
Comment (rawlingsj) [inline Makefile:195]: shall we do the same for java 17 to keep them consistent?

**Comment by rawlingsj** [comment]
@tuananh thanks for the PR, were you able look look at some of the comments?  We'll look to merge soon!

**Comment by tuananh** [comment]
> @tuananh thanks for the PR, were you able look look at some of the comments? We'll look to merge soon!

I'm gonna wrap this up this weekend

File: openjdk-17.yaml [hunk 3]
```diff
@@ -120,6 +120,39 @@ subpackages:
...
+                  mv "$_FROM_ROOT/bin/$i" "$_TO_ROOT/bin/$i"
+          done
+
+          mv "$_FROM_ROOT/legal"      "$_TO_ROOT"
+          mv "$_FROM_ROOT/conf"       "$_TO_ROOT"
+          mv "$_FROM_ROOT/release"    "$_TO_ROOT"
+          cp "./ASSEMBLY_EXCEPTION"   "$_TO_ROOT"
+          cp "./LICENSE"              "$_TO_ROOT"
+          cp "./README.md"            "$_TO_ROOT"
+    description: "Oracle OpenJDK 17 (JRE headless)"
```
Comment (luhring) [inline openjdk-17.yaml:151]: Could you remove "Oracle" to stay consistent with the other package descriptions?

**Review by luhring — CHANGES_REQUESTED** [review]

**Review by luhring — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

**Pull request merged**